db/migrate/20231025083234_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
```

Comment:
- [ ] Please make sure that all columns that are foreign keys should have a corresponding index as requested in the project requirements
 - It should be like:
```ruby
t.references :user, null: false, foreign_key: true, index: true
t.references :post, null: false, foreign_key: true, index: true
```